src/signals/incident-management/containers/DefaultTextsAdmin/saga.test.js

```diff
@@ -213,7 +213,7 @@ describe('/signals/incident-management/containers/DefaultTextsAdmin/saga', () =>
         .next()
         .put(
           actions.showGlobalNotification({
-            title: `0 Standaard teksten opgeslagen voor ${subcategory.value}, ${status.value}`,
+            title: `Standaard teksten bijgewerkt voor ${subcategory.value}, ${status.value}`,
             variant: VARIANT_SUCCESS,
```

Comment:
This seems strange feedback to receive if nothing is saved...

Reply:
Hoi, heb het uitgezocht maar eigenlijk is de melding nog niet zo raar. Om geen standaard teksten meer te hebben, moet een gebruiker die eerst actief weghalen. Ze zijn dan dus wel gewijzigd en er is wel iets gesaved.

Reply:
I have not changed the code. If you're in agreement with me, could you please approve the PR?